store/sqlstore/upgrade.go

```diff
@@ -478,7 +477,7 @@ func UpgradeDatabaseToVersion49(sqlStore SqlStore) {
 		defaultTimezone := timezones.DefaultUserTimezone()
 		defaultTimezoneValue, err := json.Marshal(defaultTimezone)
 		if err != nil {
-			mlog.Critical(fmt.Sprint(err))
+			mlog.Critical("critical", mlog.Err(err))
```

Comment:
Same question here.

Reply:
changed to mlog.Critical(err.Error()). Goal of the ticket is to remove Sprint formatting.